### PR TITLE
Semeniuk task 1

### PR DESCRIPTION
**Pull request by MikeSem**

### CODE REVIEW

**Comment by MikeSem** [comment]
Finished first task

File: myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var btn = document.querySelector('button');
```
Comment (MishaSavchak) [inline myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js]: need use .ID selectors

File: myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+var textArea = document.querySelector('textarea');
```
Comment (MishaSavchak) [inline myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js]: use single var pattern

File: myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js (hunk 1 above)
Comment (MishaSavchak) [inline myhailo.semeniuk/Lectures/Tasks/Task1 - Basics of JS/script.js]: please move logic into functions

**Comment by MishaSavchak** [comment]
Cyrillic does not work

**Pull request merged**